### PR TITLE
Add: Blog Articles for this month

### LINKED ISSUE #77: Blog update for 2nd October 2k17
- [ ] Add Piston Article 
- [ ] Add CompSoc Article
- [ ] Add Diode article

Blog articles are accessible in the [Drive Link](https://drive.google.com/drive/folders/0BxuVe1f_hNs2a21SQlliUXFLSXc?usp=sharing) here!

Make sure you don't forget about this! Any confusions feel free to contact @adwait1-G kv krishna or anirudh 

Issue is assigned to Adil!

### PR DESCRIPTION
**Pull request by adilashique**
Closes #77

### CODE REVIEW

**Comment by adilashique** [comment]
Okay so htmlproofer is giving an error. The images are present in the directories but it's showing an error anyways. Not really sure how to fix that. The images and the site work perfectly locally. 
Guru.

File: blog_src/_posts/2017-10-2-fingerprint-recognition.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+tags:
+- fingerprint
+- sensors
+- fingerprint sensor
+categories:
+- Diode
+github_username: 'manan-sharma'
+---
+
+# Fingerprint Recognition
```
Comment (salman-bhai) [inline blog_src/_posts/2017-10-2-fingerprint-recognition.md]: No need for separate header. Load in local and check, header will come twice as default header is read from article name!

File: blog_src/_posts/2017-10-02-chatbot.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Chat
+- Bots
+- Language Processing
+- IoT
+categories:
+- Compsoc
+github_username: 'adilashique'
+---
+﻿
+# The Intriguing World of Chatbots
```
Comment (salman-bhai) [inline blog_src/_posts/2017-10-02-chatbot.md]: No need for separate header. Load in local and check, header will come twice as default header is read from article name!

File: blog_src/_posts/2017-10-02-biohacking.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- hacking
+- biological hacking
+- superhuman
+- cyborgs
+categories:
+- Piston
+github_username: 'andydandyy'
+---
+
+# Biohacking
```
Comment (salman-bhai) [inline blog_src/_posts/2017-10-02-biohacking.md]: No need for separate header. Load in local and check, header will come twice as default header is read from article name!

**Pull request merged**